Command/SystemInfoDumpCommand.php

```diff
@@ -8,25 +8,55 @@
  */
 namespace EzSystems\EzSupportToolsBundle\Command;
 
+use EzSystems\EzSupportToolsBundle\SystemInfo\Collector\SystemInfoCollector;
```

Comment:
Yep, good to hear. I have updated https://jira.ez.no/browse/EZP-25647 accordingly.